charts/dapr/README.md

```diff
@@ -167,7 +167,9 @@ The Helm chart has the follow configuration options that can be supplied:
 | `dapr_sidecar_injector.image.name`        | Docker image name for Dapr runtime sidecar to inject into an application (`global.registry/dapr_sidecar_injector.image.name`) | `daprd`|
 | `dapr_sidecar_injector.injectorImage.name` | Docker image name for sidecar injector service (`global.registry/dapr_sidecar_injector.injectorImage.name`) | `dapr`|
 | `dapr_sidecar_injector.webhookFailurePolicy` | Failure policy for the sidecar injector                              | `Ignore`                |
-| `dapr_sidecar_injector.runAsNonRoot`      | Boolean value for `securityContext.runAsNonRoot`. You may have to set this to `false` when running in Minikube | `true` |
+| `dapr_sidecar_injector.runAsNonRoot`      | Boolean value for `securityContext.runAsNonRoot` for the Sidecar Injector container itself. You may have to set this to `false` when running in Minikube | `true` |
+| `dapr_sidecar_injector.sidecarRunAsNonRoot` | When this boolean value is true (the default), the injected sidecar containers have `runAsRoot: true`. You may have to set this to `false` when running Minikube | `true` |
```

Comment:
I think no need sidecar prefix as it belongs to `dapr_sidecar_injector`.

Reply:
The prefix is in place for all other variables in the tables, so I added it for consistency.

Reply:
Oh I see what you mean now, sorry.

The `sidecar` prefix such as `sidecarRunAsNonRoot` is there because there is already a `runAsNonRoot` which is about the injector service itself.

- `runAsNonRoot` -> Controls the injector service itself
- `sidecarRunAsNonRoot` -> Controls the injected sidecars